src/python_testing/TC_RVCRUNM_2_2.py

```diff
@@ -113,9 +125,16 @@ async def test_TC_RVCRUNM_2_2(self):
         # Starting the test steps
         self.print_step(1, "Commissioning, already done")
 
-        self.print_instruction(2, "Manually put the device in a RVC Run Mode cluster mode with "
-                                  "the Idle(0x4000) mode tag and in a device state that allows changing to either "
-                                  "of these modes: %i, %i" % (self.mode_a, self.mode_b))
+        # Ensure that the device is in the correct state
+        if self.is_ci:
+            self.write_to_app_pipe('{"Name": "Reset"}')
+            sleep(wait_time)
+
+        self.print_step(2, "Manually put the device in a RVC Run Mode cluster mode with "
+                           "the Idle(0x4000) mode tag and in a device state that allows changing to either "
+                           "of these modes: %i, %i" % (self.mode_a, self.mode_b))
+        if not self.is_ci:
+            input("Press Enter when done.\n")
 
         self.print_step(3, "Read the RvcRunMode SupportedModes attribute")
```

Comment:
The rest of this test needs to be updated due to this test plan change: https://github.com/CHIP-Specifications/chip-test-plans/pull/3924

Reply:
I am planning to do this change is a separate PR. This should keep this PR easy to review as it does not change the tests, it only automates them for running with the rvc-app.